api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -31,7 +31,7 @@
  * <p>A ConfigSource will get picked up via the
  * {@link java.util.ServiceLoader} mechanism and must get registered via
  * META-INF/services/javax.config.spi.ConfigSource</p>
- * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
+ * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}. << How?
```

Comment:
@Emily-Jiang I'm closing this PR - question answered. Please don't forget to improve the javadoc ;-)